deploy/charts/external-secrets/values.yaml

```diff
@@ -372,8 +372,8 @@ webhook:
   # -- Pod disruption budget - for more details see https://kubernetes.io/docs/concepts/workloads/pods/disruptions/
   podDisruptionBudget:
     enabled: false
-    minAvailable: 1
-    # maxUnavailable: 1
+    minAvailable: 1 # @schema type:[integer, string]
```

Comment:
try putting this on the TOP of this line. Then it should be okay.

Reply:
@Skarlso Hello there :) those annotations only works when they are in the side. When putting them on top, not working at all

Reply:
Because of fixinf this and TOCs i added exactly 4 spaces :) is it fine?

Reply:
unit tests failing, but I think it doesn't matter for the purpose of this changes as no merging code, right? :)